README.md

```diff
@@ -1,70 +1,179 @@
 
-You don't have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn't feel obligated to use this feature. However we understand that this tool wouldn't be useful if you couldn't customize it when you are ready for it.
+- **Calculator Using React**
+- **Use State Hook**
+- **Quotes from API**
 
-## Learn More
+<!-- LIVE DEMO -->
 
-You can learn more in the [Create React App documentation](https://facebook.github.io/create-react-app/docs/getting-started).
+<!-- - No live demo available yet -->
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-To learn React, check out the [React documentation](https://reactjs.org/).
+<!-- GETTING STARTED -->
 
-### Code Splitting
+## 💻 Getting Started <a name="getting-started"></a>
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/code-splitting](https://facebook.github.io/create-react-app/docs/code-splitting)
+To get a local copy up and running, follow these steps.
 
-### Analyzing the Bundle Size
+### Prerequisites
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size](https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size)
+- A web browser
+- A code editor
+- A terminal
 
-### Making a Progressive Web App
+### Setup
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app](https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app)
+Clone this repository to your desired folder:
 
-### Advanced Configuration
+```sh
+  git clone git@github.com:jlberbesi/Math-magicians.git
+```
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/advanced-configuration](https://facebook.github.io/create-react-app/docs/advanced-configuration)
+### Install
+
+Install this project with:
+
+```sh
+  cd math-magicians
+  npm install
+```
+
+### Usage
+
+To run the project on the webpack dev server, execute the following command:
+
+```sh
+  npm start
+```
+
+### Run tests
+
+To run tests, run the following command:
+
+
+```sh
+  npm test
+```
 
 ### Deployment
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/deployment](https://facebook.github.io/create-react-app/docs/deployment)
+```sh
+  npm run deploy
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Author <a name="authors"></a>
+
+👤 José Luis Berbesi
+
+- GitHub: [@jlberbesi](https://github.com/jlberbesi)
+- Twitter: [@imberbesi](https://twitter.com/jlberbesi)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/jlberbesi/)
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- **Add popup window for reservations 😁**
+- **Add Episodes section 👌**
+- **Add more characters 😎**
+
+## Walkthrough <a name="walkthrough"></a>
+
+You can watch the video description of the project [here](https://---).
```

Comment:
- [ ] This link is currently broken, I kindly suggest you provided the appropriate link. it is unprofessional to leave broken links in your readme file 👍 
- [ ] THis section is also not a mandatory section so you can remove it. make sure to remove it from the headers at the top of your readme file as all